crowbar_framework/app/models/api/node.rb

```diff
@@ -177,6 +177,14 @@ def disable_pre_upgrade_attribute_for(name)
       end
     end
 
+    # change the value of rabbitmq clustering
+    # (default for SOC8 is true, but false for SOC7)
+    def update_rabbitmq_config(new_value)
```

Comment:
This needs to be executed in all the cluster nodes that have rabbitmq as currently only is executed on the first one

Reply:
done